tools/explorer/lib/model.py

```diff
@@ -27,34 +29,41 @@ class StandardField(object):
         namespace_name: a field's defined namespace as a string.
         standard_field_name: the un-incremented name of the field as a string.
           must be lower-case and properly formatted.
+        increment: a field's enumerated value suffixed onto the field name.
     Attributes:
         namespace: the name of the namespace as a string
         name: the field name as a string.
+        increment: a field's enumerated value suffixed onto the field name.
     returns: An instance of the StandardField class.
   """
 
-  def __init__(self, namespace_name: str, standard_field_name: str):
+  def __init__(self,
+               namespace_name: str,
+               standard_field_name: str,
+               increment: str = ''):
     super().__init__()
-    if not FIELD_CHARACTER_REGEX.match(standard_field_name):
+    if not FQ_FIELD_NAME.match(standard_field_name + increment):
```

Comment:
how would you detect now that it is incorrectly formatted?

Reply:
Added tests.